### ISSUE TITLE
Tests have undocumented matplotlib dependency

### ISSUE THREAD

**Issue by dham** [OceanPARCELS/parcels#148]
Running py.test in the top level directory results in examples fails which are caused by a failure to import matplotlib. It turns out matplotlib is not in `requirements.txt`.

**Comment by erikvansebille** [comment]
Parcels should fail gracefully if matplotlib is not installed (it is only necessary for the plotting; Parcels should still be runnable without it).

What's the error message you're getting? On which command?

**Comment by dham** [comment]
for example:

```python
Notebook execution failed
Cell 1: no description

Input:
from parcels import Grid, ParticleSet, JITParticle
from parcels import AdvectionRK4
from datetime import timedelta as delta
from scripts import plotTrajectoriesFile
import math

Traceback:

ImportErrorTraceback (most recent call last)
<ipython-input-2-0465ed1b62bb> in <module>()
      2 from parcels import AdvectionRK4
      3 from datetime import timedelta as delta
----> 4 from scripts import plotTrajectoriesFile
      5 import math

/Users/dham/src/parcels/src/parcels/scripts/__init__.py in <module>()
----> 1 from scripts.plotParticles import *  # NOQA get flake8 to ignore unused import.

/Users/dham/src/parcels/src/parcels/scripts/plotParticles.py in <module>()
      2 from netCDF4 import Dataset
      3 import numpy as np
----> 4 import matplotlib.pyplot as plt
      5 from argparse import ArgumentParser
      6 import matplotlib.animation as animation

ImportError: No module named matplotlib.pyplot
```

**Comment by dham** [comment]
this is just by running `py.test` in the top parcels directory.

**Comment by erikvansebille** [comment]
Fixed in #153
